File: backend/src/Modules/CarRental/Domain/Values/Location.cs
namespace CarRental.Domain.Values;

public class Location
{
    public double Latitude { get; private set; }

    public double Longitude { get; private set; }

    public string Address { get; private set; }

    public Location(double latitude, double longitude, string address)
    {
        Latitude = latitude;
        Longitude = longitude;
        Address = address;
    }
}

File: mobile-app/app/src/main/java/com/example/easydrive/network/model/Location.java
package com.example.easydrive.network.model;

import com.google.gson.annotations.SerializedName;

public class Location {
    @SerializedName("latitude")
    private double latitude;
    
    @SerializedName("longitude")
    private double longitude;
    
    @SerializedName("address")
    private String address;

    // Getters and setters
    public double getLatitude() { return latitude; }
    public void setLatitude(double latitude) { this.latitude = latitude; }
    public double getLongitude() { return longitude; }
    public void setLongitude(double longitude) { this.longitude = longitude; }
    public String getAddress() { return address; }
    public void setAddress(String address) { this.address = address; }
} 